dockers/docker-config-engine-bullseye/Dockerfile.j2

```diff
@@ -23,6 +23,9 @@ RUN apt-get install -y        \
 RUN pip3 install pyangbind==0.8.1
 RUN pip3 uninstall -y enum34
 
+# Install python-redis
+RUN pip3 install redis==4.5.4
```

Comment:
>==4.5.4 [](http://example.com/codeflow?start=22&length=7)

@xumia, do we need to specify the version?

Reply:
> > ==4.5.4
> 
> @xumia, do we need to specify the version?
If python3-redis installed (redis=3.5.3 currently), then need to specify the version, or use pip3 install --upgrade redis. If the version not specified, then the installation will be skipped, use 3.5.3.

If we need the version 4.5.4 exactly, then "pip3 install redis==4.5.4" is good.
If we want to use the latest version, and at least 4.5.4, use "pip3 install redis>=4.5.4", prefer to auto version upgrade.

Reply:
Hi, we need exactly 4.5.4, because swsssdk install 4.5.4:

https://github.com/sonic-net/sonic-py-swsssdk/blob/master/setup.py
dependencies = [
    'redis==4.5.4;python_version >= "3.0"',
    'redis>=3.5.3;python_version < "3.0"',
    'redis-dump-load',
]